spec/webdrivers/chrome/driver_remote_version_finder_spec.cr

```diff
@@ -2,9 +2,9 @@ require "../../spec_helper"
 
 module Webdrivers::Chrome
   describe DriverRemoteVersionFinder do
-    before_each do
-      finder = DriverRemoteVersionFinder.new(Common.driver_directory)
-      File.delete(finder.cache_path) if File.exists?(finder.cache_path)
+    Spec.after_each do
```

Comment:
after_each is not great to use cuz it only runs if the test passes. So if one fails for a good reason, it can create a cascade of false negatives cuz the after block didn't run

Reply:
oh, I didn't know that. before_each also doesn't work because it just leaves the files laying around... Does `around_each` work better in this case?

Reply:
Alright, so I tested around_each by adding a `true.should eq(false)` in the middle of the spec, and it cleaned up the file.

Reply:
Yep, around_each is the better way 👍... but I'm wondering if we should point the testing cache path to a random tmp dir and not worry about cleaning up after. The only problem that occurs to me about cleaning after instead of before is if we use the same cache as normal and you have an existing driver in the directory, it could mess with tests as well.

Reply:
It seems like we name the chromedriver differently in tests (at least that's what it looks like judging by just this PR and not diving back into the code), so maybe it isn't an issue?

Reply:
I think since I've got a green CI here, I'll just roll with this. If it does become an issue, maybe someone else can dive in on this.